src/main/java/net/mcreator/element/types/Particle.java

```diff
@@ -62,6 +62,9 @@
 	public boolean alwaysShow;
 	public boolean emissiveRendering;
 
+	public boolean lockRot;

```

Comment:
1. Use more understandable names, lockRotation, for example
2. Specify default value. Existing elements should likely have this value at true